libs/blocks/quiz-entry/utils.js

```diff
@@ -7,14 +7,25 @@ export async function fetchJson(path) {
 
 export async function getQuizJson(path) {
   try {
-    const [questions, strings] = await Promise.all(
-      [fetchJson(`${path}questions.json`), fetchJson(`${path}strings.json`)],
-    );
-    return [questions, strings];
+    // Fetch required files
+    const [questions, strings] = await Promise.all([
+      fetchJson(`${path}questions.json`),
+      fetchJson(`${path}strings.json`),
+    ]);
+
+    // Try to fetch optional file, results.json
+    let results = [];
+    try {
+      results = await fetchJson(`${path}results.json`);
+    } catch (ex) {
+      window.lana?.log(`INFO: results.json not found or couldn't be fetched: ${ex}`);
```

Comment:
Please refer to @robert-bogos's note on LANA logs categorization:
> It's important to include these tags as they help us filter logs in Splunk and find real errors before they become CSOs.
This is an example of adding tags to a log: window.lana.log('Failed to fetch suggestions', { tags: 'errorType=info,module=quiz-entry' }) 
where the tags can be:
errorType=info - network issues
errorType=warn - authoring related misconfigs or similar
errorType=error - actual error ( ex. cannot read Y of undefind )
module=<module>- extra tag to help centralise all logs related to a specific module

Reply:
Thanks!